guides/source/layout.html.erb

```diff
@@ -90,59 +102,70 @@
               <% end %>
             </select>
           </li>
-      </ul>
+        </ul>
       </nav>
     </div>
   </header>
+
   <hr class="hide" />
 
-  <section id="feature">
-    <div class="wrapper">
-      <%= yield :header_section %>
+  <main id="main">
+    <article>
+      <header id="feature">
+        <div class="wrapper">
+          <%= yield :header_section %>
 
-      <%= yield :index_section %>
+          <%= yield :index_section %>
+        </div>
+      </header>
 
-      <hr>
-    </div>
-  </section>
+      <div class="wrapper">
+        <div id="column-main">
+          <section id="article-body">
```

Comment:
For the guides index it would have the entire index in there, which kinda makes sense. Would we label the region? Or just as a way to jump there? It is harder if we need to label it, but still possible.

Reply:
I mean, add a region to the article body wrapper so that you can easily jump to it using quick navigation commands between regions. But only in the article body, not at the general level, because in the guide index there should be no such region.

We should probably create an extra wrapper in `markdown.rb`.

Reply:
The change is

```ruby
doc.wrap("<section role='region'>")
```

on line 134, and we need some changes to the anchors checking then. However, I can not make this change because after rebasing I can no longer run `rake guides:generate` probably because it expects the latest `rails` version which I cannot install on the box I am running this PR on. I therefore would like to defer this change for now.